status-im/status-react
# New intro screen

fixes: https://github.com/status-im/status-mobile/issues/15014

This pr implements the basic structure of the new "intro" page.

The carousel implemented is only a quick solution with no animation, I will create a separate issue to address this and also add blur effect to the buttons. Currently we want to get this flow started so other devs work can begin as well.

<img width="234" alt="Screenshot 2023-02-17 at 23 17 42" src="https://user-images.githubusercontent.com/22799766/219816128-501d0301-2586-46b3-9818-4f1b2a3e2292.png">
<img width="218" alt="Screenshot 2023-02-17 at 23 17 38" src="https://user-images.githubusercontent.com/22799766/219816132-129024df-7e08-4a83-a4c6-ee2bcd596b04.png">


<img width="233" alt="Screenshot 2023-02-17 at 23 17 40" src="https://user-images.githubusercontent.com/22799766/219816131-29f11080-3171-45a3-9887-b03766d04078.png">

<img width="214" alt="Screenshot 2023-02-17 at 23 17 53" src="https://user-images.githubusercontent.com/22799766/219816123-cecfad76-235d-400d-b367-eb95f09924d0.png">


Currently both buttons will just direct the user to the next page that would have been on the flow anyway.
i.e
no matter what they select they will be brought to:

<img width="209" alt="Screenshot 2023-02-17 at 23 18 14" src="https://user-images.githubusercontent.com/22799766/219815984-30505f8e-a33d-4994-82d8-afb2e3559b5c.png">

To test:
Open up the application without any saved data, you should see the new opening screen
